tests/lib.rs

```diff
@@ -4,6 +4,12 @@
 #[cfg(feature = "std")]
 extern crate core;
 
+// only import macros with std
+#[cfg(feature = "std")]
+#[macro_use]
```

Comment:
Wasn't there an `#[if()]` clause or sth?

Reply:
Right! `cfg_attr` it is.